tests/plugins/test_tag_by_labels.py

```diff
@@ -63,20 +69,33 @@ def test_tag_by_labels_plugin(tmpdir):
         workflow,
         [{
             'name': TagByLabelsPlugin.key,
+            'args': args,
         }]
     )
     output = runner.run()
     assert TagByLabelsPlugin.key in output.keys()
-    assert len(workflow.tag_conf.images) == 4
+
     images = [i.to_str() for i in workflow.tag_conf.images]
     primary_images = [i.to_str() for i in workflow.tag_conf.primary_images]
     unique_images = [i.to_str() for i in workflow.tag_conf.unique_images]
+
+    if args.get('unique_tag_only'):
+        assert len(workflow.tag_conf.images) == 1
+        assert len(primary_images) == 0
+
+    else:
+        assert len(workflow.tag_conf.images) == 4
+        assert len(primary_images) == 3
+
+        assert ("%s:%s-%s" % (TEST_IMAGE, version, release)) in images
+        assert ("%s:%s" % (TEST_IMAGE, version)) in images
+        assert ("%s:latest" % (TEST_IMAGE, )) in images
```

Comment:
assertions done on lines 99 and 100 are always performed. It's not a timestamp, but it's a unique tag which in this case serves the same purpose.